app/workers/share_worker.rb

```diff
@@ -11,6 +11,15 @@ def perform(user_id, answer_id, service)
     user_service = User.find(user_id).services.find_by(type: service_type)
 
     user_service.post(Answer.find(answer_id))
+  rescue ActiveRecord::RecordNotFound
+    # The question to be posted was deleted
```

Comment:
please add some log messages (and maybe some `NewRelic::Agent.notice_error(e)`s) to these so we know how often this happens